### PR TITLE
Fixes #34910 - Replace pf3 OverlayTrigger

### PR DESCRIPTION
**Pull request by MariaAga**
depends on https://github.com/theforeman/foreman_templates/pull/161

### CODE REVIEW

**Comment by ofedoren** [comment]
Thanks, @MariaAga, I guess this needs to be rebased. Also, could you please use a separate Redmine ticket as with other plugins?
pretty-please

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 language: node_js
 node_js:
-  - '8'
   - '10'
   - '12'
```
Comment (adamruzicka) [inline .travis.yml:4]: While you're touching this, could o

**Comment by MariaAga** [comment]
Thanks for the reminder :) Separated and rebased.
tumblr_ow33hbRXZO1v4a8wfo5_r1_540

**Review by ofedoren — APPROVED** [review]
Thanks, @MariaAga, LGTM and works as expected :)

**Pull request merged**